agent/api/container/container.go

```diff
@@ -269,6 +272,7 @@ type Secret struct {
 	ContainerPath string `json:"containerPath"`
 	Type          string `json:"type"`
 	Provider      string `json:"provider"`
+	Target        string `json:"target"`
```

Comment:
sorry that i forgot.. i think you need to bump state file version for this

Reply:
Done!